### PR TITLE
Upgrade Gradle Wrapper to 6.2.2 and build scan to 3.1.1

### PR DESCRIPTION
**Pull request by gerritsangel**
Gradle 5 was not compatible with OpenJDK 13 anymore.

While upgrading to Gradle 6, I also had to upgrade the Gradle buildscan to version 3.1.1.

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1 +1,5 @@
+plugins {
+  id "com.gradle.enterprise" version "3.1.1"
+}
```
Comment (abhinayagarwal) [inline settings.gradle:3]: Why do we want to use this plugin?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,5 @@
-#Tue Nov 13 11:59:28 CET 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-bin.zip
```
Comment (abhinayagarwal) [inline gradle/wrapper/gradle-wrapper.properties:3]: Better to update to Gradle 6.3 which supports Java 14.

File: settings.gradle (hunk 1 above)
Comment (koppor) [reply settings.gradle:3]: This is the update to "Gradle Buildscan". Source: https://guides.gradle.org/creating-build-scans/#enable_build_scans_on_all_builds_of_your_project

> For Gradle 2.x-5.x, you need to apply the com.gradle.build-scan plugin to your root build script. **For Gradle 6.0, you need to apply the `com.gradle.enterprise` plugin to your settings script.**.

Thus, either the build scan has to be disabled or that plugin has to be used.

Comment (koppor) [reply settings.gradle:3]: I went for disabling in #79

**Comment by gerritsangel** [comment]
Closing this for https://github.com/openjfx/javafx-gradle-plugin/pull/79.

**Pull request closed without merging**